src/main/scala/io/iohk/ethereum/domain/BlockHeader.scala

```diff
@@ -57,11 +51,8 @@ case class BlockHeader(
 
   override def toString: String = {
     val (treasuryOptOutString: String, checkpointString: String) = extraFields match {
```

Comment:
shouldn't we also remove or rename `treasuryOptOutString` here ?

Reply:
agreed, initially I wanted to have a marker but it does not make sense in this context